schemas/api/appIdentifier.schema.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What is the impact of the same types being defined as TypeScript files and JSON schema files? Which is the golden source? Are the TypeScript types now generated from these, or are they maintained separately? If so, how do we ensure consistency of changes?

Reply:
The schemas were generated from the typescript types via quicktype. The schemas don't carry functions of course. 

I think the only impact is positive. @robmoffat for example was keen to have these for use in FDC3 for Web Browsers proposals (where proxying DA over postMessage)

Reply:
So long as they are generated, I agree the only impact is positive.

Reply:
I note the script for that generation isn't included in the PR, it can probably be added to the package.json so it can be repeated/automated.

Reply:
Generation script now included in PR - it gets 95% of the job done, but there are a couple of manual steps due to Quicktypes limitations on files with functions that return promises.